nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-frontend/src/main/nifi/src/app/pages/provenance/ui/provenance-event-listing/provenance-event-table/provenance-event-table.component.ts

```diff
@@ -77,16 +86,69 @@ export class ProvenanceEventTable implements AfterViewInit {
             if (filterTerm?.length > 0) {
                 const filterColumn = this.filterForm.get('filterColumn')?.value;
                 this.applyFilter(filterTerm, filterColumn);
+            } else {
+                this.resetPaginator();
             }
         }
     }
     @Input() oldestEventAvailable!: string;
+    @Input() timeOffset!: number;
     @Input() resultsMessage!: string;
     @Input() hasRequest!: boolean;
+    @Input() loading!: boolean;
+    @Input() loadedTimestamp!: string;
+    @Input() set lineage$(lineage$: Observable<Lineage | null>) {
+        this.provenanceLineage$ = lineage$.pipe(
+            tap((lineage) => {
+                let minMillis: number = -1;
+                let maxMillis: number = -1;
+
+                lineage?.results.nodes.forEach((node) => {
+                    // ensure this event has an event time
+                    if (minMillis < 0 || minMillis > node.millis) {
+                        minMillis = node.millis;
+                        // minTimestamp = node.timestamp;
```

Comment:
is the minTimestamp not needed? can we remove this?

Reply:
Yup!